include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -1906,7 +1919,7 @@ MatrixFree<dim, Number, VectorizedArrayType>::initialize_indices(
     additional_data.cell_vectorization_categories_strict,
     do_face_integrals,
     additional_data.mapping_update_flags_inner_faces != update_default,
-    additional_data.overlap_communication_computation,
+    overlap_communication_computation,
```

Comment:
It looks like this change is now done unconditionally when we detect `FE_Nothing`, not just when we have face integrals. I thought that the problem was only in the latter case?